webapp/pages/post/_id/_slug/index.vue

```diff
@@ -200,8 +200,8 @@ export default {
 
     .ds-card-image {
       img {
-        max-height: 710px;
-        object-fit: cover;
+        max-height: 2000px;
```

Comment:
> [<img alt="alina-beck" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/alina-beck) **Authored by [alina-beck](https://github.com/alina-beck)**
_<time datetime="2019-10-14T16:55:59Z" title="Monday, October 14th 2019, 6:55:59 pm +02:00">Oct 14, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -200,7 +200,7 @@ export default {
 
     .ds-card-image {
       img {
-        max-height: 710px;
+        max-height: 2000px;
         object-fit: cover;
```


```suggestion
        object-fit: contain;
```

I would go with `contain` here – same as I suggested above. This means that the whole image is visible (and white borders are added to the sides if it is too narrow, i.e. if it exceeds a ratio of 2000px / 300px which will rarely be the case). `cover` on the other hand means that the whole area is filled but if the image is too long it gets cropped, which, as I understood from today's meeting, we do not want. Oder? ;)